components/wifi/esp8266-driver/ESP8266Interface.cpp

```diff
@@ -71,6 +71,7 @@ ESP8266Interface::ESP8266Interface()
       _oob_event_id(0),
       _connect_event_id(0)
 {
+    mbed_trace_init();
```

Comment:
I assume this should have been removed.

Reply:
@VeijoPesonen  , it is tricky to have `scanf` do the job for us here, because we search for strings of different length and `scanf` will return as soon as it matches anything (like - one letter is good enough to match a string).
`recv` [has a workaround for this](https://github.com/ARMmbed/mbed-os/blob/master/platform/source/ATCmdParser.cpp#L310-L312) so it is better suited for usage in `_oob_connection_status()`.